pages/cloud-providers.html

```diff
@@ -28,33 +28,18 @@
 
         <div class="card flow-small">
             <div class="card__title">
-                Chère de Prince
+                La Bécasse
```

Comment:
I would keep "Chère de Prince" as "La Bécasse" is only the nickname of the person managing Chère de Prince (see https://cheredeprince.net "Qui suis-je ?")

Reply:
Actually, I got it from there only, but yeah, I agree. I will revert the same.